main/src/ui/util/helper.vala

```diff
@@ -381,7 +381,9 @@ public static string parse_add_markup(string s_, string? highlight_word, bool pa
                     int start, end;
                     match_info.fetch_pos(2, out start, out end);
                     return parse_add_markup(s[0:start-1], highlight_word, parse_links, parse_text_markup, already_escaped) +
-                        s[start-1:start] + @"<$(convenience_tag[i])>" + s[start:end] + @"</$(convenience_tag[i])>" + s[end:end+1] +
+                        "<span color='#999'>" +  s[start-1:start] + "</span>" +
```

Comment:
Please don't override colors from the theme. While unlikely, it's possible that users might want to use a theme where fg text has a different color like green. In that case, the grey symbols will look strange. I recommend it either use `alpha='50%'` or `font_weight='ultralight'` (you can adjust the values to your needs). Markup reference: https://developer.gnome.org/pango/stable/pango-Markup.html